### PR TITLE
Add Grafana configuration information

### PR DESCRIPTION
**Pull request by witekest**
# Description

Add steps for manual Grafana configuration

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3 https://github.com/uyuni-project/uyuni-docs/pull/1952
- [ ] Manager-4.2

# Links

Fixes SUSE/spacewalk#19714

### CODE REVIEW

**Comment by witekest** [comment]
I'm not sure if the screenshot is added in a right way. When building locally it was not rendered.
Apart from that it also breaks the numbering of the list.

File: modules/administration/pages/monitoring.adoc [hunk 1]
```diff
@@ -234,8 +234,22 @@ zypper in grafana
...
 ----
-. Check that the Grafana interface is loading correctly.
-    In your browser, navigate to the URL of the server where Grafana is installed, on port 3000 (for example, [literal]``http://example.com:3000``).
+. In your browser, navigate to the URL of the server where Grafana is installed, on port 3000 (for example, [literal]``http://example.com:3000``).
+. On the login page, enter ``admin`` for username and password.
+. Click btn:[Log in]. If login is successful, then you will see a prompt to change the password.
+. Click btn:[OK] on the prompt, then change your password.
+. Move your cursor to the cog icon on the side menu which will show the configuration options.
+
+image::grafana_sidemenu_datasource.png[scaledwidth=40%]
```
Comment (0rnela) [inline modules/administration/pages/monitoring.adoc:243]:
```suggestion
+
image::grafana_sidemenu_datasource.png[scaledwidth=40%]
```

File: modules/administration/pages/monitoring.adoc [hunk 2]
```diff
@@ -234,8 +234,22 @@ zypper in grafana
...
+. Click on btn:[Data sources].
+. Click btn:[Add data source] to see a list of all supported data sources.
+. Choose the Prometheus data source.
+. Make sure to specify the correct URL of the Prometheus server.
+. Click btn:[Save & test].
+. To import a dashboard click the btn:[+] icon in the side menu, and then click btn:[Import].
+. For {productname} server overview load the dashboard ID: ``17569``.
+. For {productname} clients overview load the dashboard ID: ``17570``.
 
 image::monitoring_grafana_example.png[scaledwidth=80%]
```
Comment (0rnela) [inline modules/administration/pages/monitoring.adoc:254]:
```suggestion
+
image::monitoring_grafana_example.png[scaledwidth=80%]
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

File: modules/administration/pages/monitoring.adoc [hunk 3]
```diff
@@ -234,8 +234,22 @@ zypper in grafana
 ----
 systemctl enable --now grafana-server
 ----
-. Check that the Grafana interface is loading correctly.
-    In your browser, navigate to the URL of the server where Grafana is installed, on port 3000 (for example, [literal]``http://example.com:3000``).
+. In your browser, navigate to the URL of the server where Grafana is installed, on port 3000 (for example, [literal]``http://example.com:3000``).
+. On the login page, enter ``admin`` for username and password.
+. Click btn:[Log in]. If login is successful, then you will see a prompt to change the password.
```
Comment (0rnela) [inline modules/administration/pages/monitoring.adoc:239]:
```suggestion
. Click btn:[Log in]. 
    If login is successful, then you will see a prompt to change the password.
```

File: modules/administration/pages/monitoring.adoc [hunk 4]
```diff
@@ -234,8 +234,22 @@ zypper in grafana
...
+. Click btn:[OK] on the prompt, then change your password.
+. Move your cursor to the cog icon on the side menu which will show the configuration options.
+
```
Comment (0rnela) [inline modules/administration/pages/monitoring.adoc:242]:
```suggestion
```

File: modules/administration/pages/monitoring.adoc [hunk 5]
```diff
@@ -234,8 +234,22 @@ zypper in grafana
...
+image::grafana_sidemenu_datasource.png[scaledwidth=40%]
+
+. Click on btn:[Data sources].
+. Click btn:[Add data source] to see a list of all supported data sources.
+. Choose the Prometheus data source.
+. Make sure to specify the correct URL of the Prometheus server.
+. Click btn:[Save & test].
+. To import a dashboard click the btn:[+] icon in the side menu, and then click btn:[Import].
+. For {productname} server overview load the dashboard ID: ``17569``.
+. For {productname} clients overview load the dashboard ID: ``17570``.
```
Comment (0rnela) [inline modules/administration/pages/monitoring.adoc:253]:
```suggestion
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

**Review by keichwa — APPROVED** [review]

**Review by 0rnela — APPROVED** [review]

**Pull request merged**